lib/jobs/linux-catalog.js

```diff
@@ -65,10 +67,22 @@ function catalogJobFactory(
         });
 
         this._subscribeRespondCommands(function(data) {
+            debugger;
```

Comment:
I just see  these debug code, I have to withdraw my approve until these debug code is removed.

:-1: